tests/accidental_tests.ts

```diff
@@ -643,39 +888,43 @@ function sagittal(options: TestOptions): void {
   const notes = [
     f
       .StaveNote({ keys: ['d/4', 'f/4', 'b/4', 'b/4'], duration: '4' })
-      .addModifier(accid('accSagittal11MediumDiesisUp'), 1)
-      .addModifier(accid('accSagittal5CommaDown'), 2)
+      .addModifier(accid('\uE30A' /*accSagittal11MediumDiesisUp*/), 1)
+      .addModifier(accid('\uE303' /*accSagittal5CommaDown*/), 2)
       .addModifier(accid('b'), 3)
-      .addModifier(accid('accSagittal7CommaDown'), 3),
+      .addModifier(accid('\uE305' /*accSagittal7CommaDown*/), 3),
 
     f
       .StaveNote({ keys: ['d/4', 'f/4', 'a/4', 'b/4'], duration: '4' })
-      .addModifier(accid('accSagittal35LargeDiesisDown'), 2),
+      .addModifier(accid('\uE30F' /*accSagittal35LargeDiesisDown*/), 2),
 
-    f.StaveNote({ keys: ['c/4', 'e/4', 'g/4', 'c/5'], duration: '8' }).addModifier(accid('accSagittal5CommaDown'), 1),
+    f
+      .StaveNote({ keys: ['c/4', 'e/4', 'g/4', 'c/5'], duration: '8' })
+      .addModifier(accid('\uE303' /*accSagittal5CommaDown*/), 1),
 
     f
       .StaveNote({ keys: ['c/4', 'e/4', 'g/4', 'b/4'], duration: '8' })
       .addModifier(accid('b'), 1)
-      .addModifier(accid('accSagittal7CommaDown'), 1)
-      .addModifier(accid('accSagittal11LargeDiesisDown'), 3),
+      .addModifier(accid('\uE305' /*accSagittal7CommaDown*/), 1)
+      .addModifier(accid('\uE30D' /*accSagittal11LargeDiesisDown*/), 3),
```

Comment:
In Vexflow 5 it is now possible to use all the SMuFL codes. The dictionary would be very big. @ronyeh is conisdering what to do in #23

Reply:
Ideally, our APIs could accept both character codes and SMuFL glyph names.

Once we are done with converting the v4 codebase to the v5 full SMuFL approach, we can investigate how to do this (or if it's better altogether to just use the friendly SMuFL glyph names).

Many of our APIs also support the classic VexFlow shorthands (e.g., ## instead of accidentalDoubleSharp).

Would be cool if we supported the classic shorthands + friendly SMuFL glyph names + unicode characters (which is the most direct method).

But we'll have to see how the library size is impacted if we support all the SMuFL names.

Reply:
+1 to accepting all three types. Ideally, in the VexFlow codebase, we end up using the readable names as much as possible.

The following would be nice:

```typescript
note.addModifier(smufl.accSagittal7CommaDown);
note.addModifier('\uE305');
```

Two approaches here:

1) Bundle the most common codes with VexFlow, and distribute the rest in a separate library (maybe `vexflow-ext-smufl`.) For our internal tests, we can pull it in as needed.

2) Depending on how many codes we're talking about, we could simply bundle the whole thing. This should be very compressible. Is there a list somewhere?

Reply:
I do not know if it is complete https://github.com/vexflow/vexflow/blob/main/tools/fonts/config/glyphnames.js

Reply:
Thanks Rodrigo. I crunched the numbers on that, and removing the description from each of the codepoints, that data compresses down to ~25k.

The most recent minimized compressed vexflow package is ~213k. So option 2 is about 12% which is pretty sizeable :-(

@ronyeh, what do you think about option 1?

Reply:
There is an option 3. Deliver one with and one without and let the user decide.

Reply:
Welcome back @0xfe ! We have missed your presence & feedback :-)

I'm going to move this conversation over to https://github.com/vexflow/vexflow/issues/23

That thread is now pinned, and will be used to discuss how we can best use SMuFL friendly names in our API.

Reply:
Thanks for the kind words, Ron. I've been on a slow recovery, but happy to see all the progress here. Great stuff, everyone!

Will respond back in #23.